safe/definitions/__init__.py

```diff
@@ -19,12 +19,14 @@
 from safe.definitions.hazard import *
 from safe.definitions.hazard_category import *
 from safe.definitions.hazard_classifications import *
+from safe.definitions.keyword_properties import *
```

Comment:
__[flake8]__
* [F403] 'from safe.definitions.keyword_properties import *' used; unable to detect undefined names
* [F401] 'safe.definitions.keyword_properties.*' imported but unused

<!-- Comment by [SideCI](https://sideci.com) -->